Data-Structures/Graph/Graph3.js

```diff
@@ -0,0 +1,120 @@
+class Graph {
+  constructor () {
+    this.adjacencyObject = {}
+  }
+
+  addVertex (vertex) {
+    if (!this.adjacencyObject[vertex]) this.adjacencyObject[vertex] = []
+  }
+
+  addEdge (vertex1, vertex2) {
+    if (this.adjacencyObject[vertex1] && this.adjacencyObject[vertex2]) {
+      this.adjacencyObject[vertex1].push(vertex2)
+      this.adjacencyObject[vertex2].push(vertex1)
+    } else {
+      return undefined
+    }
+  }
+
+  removeEdge (vertex1, vertex2) {
+    if (this.adjacencyObject[vertex1] && this.adjacencyObject[vertex2]) {
+      this.adjacencyObject[vertex1] = this.adjacencyObject[vertex1].filter(
+        (v) => v !== vertex2
+      )
+      this.adjacencyObject[vertex2] = this.adjacencyObject[vertex2].filter(
+        (v) => v !== vertex1
+      )
+    } else {
+      return undefined
+    }
+  }
+
+  removeVertex (vertex) {
+    if (this.adjacencyObject[vertex]) {
+      while (this.adjacencyObject[vertex].length) {
+        const adjacentVertex = this.adjacencyObject[vertex].pop()
+        this.removeEdge(vertex, adjacentVertex)
+      }
+    } else {
+      return undefined
```

Comment:
Same comment.

Reply:
@raklaptudirm 
I modified the code what you commented.
Thanks a lot raklaptudirm.
Thanks to you I learn a lot of things to contribute open source project.